op-e2e/actions/proofs/holocene_activation_test.go

```diff
@@ -26,28 +26,41 @@ func Test_ProgramAction_HoloceneActivation(gt *testing.T) {
 
 		env := helpers.NewL2FaultProofEnv(t, testCfg, helpers.NewTestParams(), helpers.NewBatcherCfg(), setHoloceneTime)
 
-		t.Log("HoloceneTime:  ", env.Sequencer.RollupCfg.HoloceneTime)
+		t.Logf("L2 Genesis Time: %d, HoloceneTime: %d ", env.Sequencer.RollupCfg.Genesis.L2Time, *env.Sequencer.RollupCfg.HoloceneTime)
 
-		// Build the L2 chain
-		blocks := []uint{1, 2}
-		targetHeadNumber := 2
-		for env.Engine.L2Chain().CurrentBlock().Number.Uint64() < uint64(targetHeadNumber) {
+		// Build the L2 chain until the Holocene activation time,
+		// which for the Execution Engine is an L2 block timestamp
+		// https://specs.optimism.io/protocol/holocene/exec-engine.html?highlight=holocene#timestamp-activation
+		for env.Engine.L2Chain().CurrentBlock().Time < *env.Sequencer.RollupCfg.HoloceneTime {
 			env.Sequencer.ActL2StartBlock(t)
 			// Send an L2 tx
 			env.Alice.L2.ActResetTxOpts(t)
 			env.Alice.L2.ActSetTxToAddr(&env.Dp.Addresses.Bob)
 			env.Alice.L2.ActMakeTx(t)
 			env.Engine.ActL2IncludeTx(env.Alice.Address())(t)
 			env.Sequencer.ActL2EndBlock(t)
+			b := env.Engine.L2Chain().GetBlockByHash(env.Sequencer.L2Unsafe().Hash)
+			t.Log("Unsafe block with timestamp %d", b.Time)
+			if b.Time() < *env.Sequencer.RollupCfg.HoloceneTime {
+				require.Equal(t, "", string(b.Extra()), "extra data should be empty before Holocene activation")
+			} else {
+				require.NotEqual(t, "", string(b.Extra()), "extra data should be NOT be empty after Holocene activation")
```

Comment:
maybe even more precisely
```suggestion
				require.Len(t, b.Extra(), 32, "extra data should be NOT be empty after Holocene activation")
```

Reply:
Made this change here, but used 9 bytes as per the spec [9197d00](https://github.com/ethereum-optimism/optimism/pull/13188/commits/9197d00d5e03170ca9a739a0f25e87b7bd4cd787)